src/axios/axios-pets.js

```diff
@@ -1,7 +1,7 @@
-import axios from 'axios'
+import axios from 'axios';
 
 const instance = axios.create({
-    baseURL: 'https://adote-um-pet-df1b0.firebaseio.com/pets.json'
+  baseURL: 'https://adote-um-pet-df1b0.firebaseio.com/pets.json',
```

Comment:
Would be better put it in a .env file, and then add it to the .gitignore, just to improve the security.